Fixed #27590 -- Allowed customizing a manifest file storage in ManifestFilesMixin. (#12187)
django/django: docs/ref/contrib/staticfiles.txt

    @import url("../admin/css/base.27e20196a850.css");

> You can store manifest file with code by subclassing

Reviewer: ```suggestion
You can specify the location of the manifest file by subclassing
```

> ``ManifestStaticFilesStorage`` and setting the ``manifest_storage`` attribute::

Reviewer: Curious: why are we discussing setting an attribute here instead of using a keyword argument? Am I missing something?

Author: Changed `attribute` to `argument`.
